docs/source/developer-guides/prs-and-reviews.rst

```diff
@@ -0,0 +1,75 @@
+Consolidation:
+==============
+
+Overarching Goals of good Review
+--------------------------------
+
+The goal of a review is to ensure that the proposed change is useful and maintainable over the long run.
+
+PR Description
+---------------
+
+- Have a meaningful title: it's easier when creating the changelog or when searching through old PR
+- Motivate why we should merge the PR: adding/changing code risks introducing a new bug. IMO one person asking for a nice to have feature doesn't qualify.
+- Explain the PR does in the description: it makes it easier/faster to review
+- Make the PR as small as possible: it's much easier to review five 200 lines PR than one 1000 lines PR
+- If a PR is known to create conflict with other active PRs, try to coordinate, and link to each other
+  - if appropriate explain desired review/merge order 
+- For complex changes that will require multiple PR, create an issue to keep track of what has been done and what's left to do.
+- Is the PR focused on a single bugfix or feature?
+  - consider moving self contained changes to separate PRs 
```

Comment:
- I know what you mean here, but I'm not sure it would be clear to a wider audience
- Instead, could you say something like, "... please propose one, and only one focused / surgical change in one PR.  It is fine to submit multiple, focused PR to address a larger issue ..."